docs/experimental/examples/oauth2/globus_app.rst

```diff
@@ -1,5 +1,202 @@
 
-Using a Globus App
-==================
+.. _using_globus_app:
 
-TODO
+.. py:currentmodule:: globus_sdk.experimental.globus_app
+
+Using a GlobusApp
+=================
+
+Programmatic communication with Globus services relies on the authorization of requests.
+Management and resolution of this authorization can become an arduous task, especially
+when a script needs to interact with different services each carrying an individual set
+of complex auth requirements. To assist with this task, this library provides a utility
+construct called a GlobusApp.
+
+A :py:class:`~GlobusApp` is a distinct object which will manage auth requirements
+(i.e. **scopes**) identified by their associated service (i.e. **resource server**).
```

Comment:
```suggestion
(i.e., **scopes**) identified by their associated service (i.e., **resource server**).
```

Does GlobusApp provide any special handling vis-a-vis session-related issues today?  I don't believe so, but want to double check, as scopes/consents are only one dimension of auth requirements.

Reply:
It allows you to provide them on explicit login requests but not in the same way that scopes are managed, I'll sstick with `i.e.` for now.

Also, I know you're right so I'll fix it but I'd like to register my disdain for the period followed by the comma in `i.e.,`